scapy/arch/windows/__init__.py

```diff
@@ -72,7 +74,7 @@ def _vbs_exec_code(code):
 
 def _vbs_get_iface_guid(devid):
     try:
-        # devid = str(int(devid) + 1)
+        devid = str(int(devid) + 1) # devid in the registry seems not aligned on WinXPSP3 with wmi's fields DeviceID and/or Index #TODO check if it is always the case and what happens when there are multiple interfaces on the machine
```

Comment:
I'm not sure about this one. I had a doubt (hence the commented code), but WinPcap (from wireshark) shows the available interfaces as "interface name [Microsoft's Packet Scheduler]", and the interface with index `devid` is named something like "Microsoft's Packet Scheduler", and the interface with index `devid + 1` is named "interface name".

So maybe the real interface used is the "Microsoft's Packet Scheduler" related to "interface name"?